recipes/kneebow/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "kneebow" %}
+{% set version = "1.0.2" %}
+{% set python_min = "3.8" %}  # Updated to reflect conda-forge minimum Python version if needed.
```

Comment:
Why choose python 3.8 here instead of the global pinning of 3.9?

Reply:
python version changed to 3.9